base/leaky_bucket.go

```diff
@@ -283,7 +283,12 @@ func (b *LeakyBucket) UUID() (string, error) {
 		return b.bucket.UUID()
 }
 
-
+func (b *LeakyBucket) CloseAndDelete() error {
+	if bucket, ok := b.bucket.(sgbucket.DeleteableBucket); ok {
+		return bucket.CloseAndDelete()
+	}
+	return nil
+}
```

Comment:
Drone failed without this cahnge:
=== RUN   TestFlush
--- FAIL: TestFlush (0.03s)
	api_test.go:254: Response status 503 (expected 200) for POST <http://localhost/db/_flush> : {"error":"Service Unavailable","reason":"Bucket does not support flush"}